react-typescript/tmpl/src/index.ts

```diff
@@ -4,7 +4,7 @@ import installExtension, { REACT_DEVELOPER_TOOLS } from 'electron-devtools-insta
 
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
-let mainWindow;
+let mainWindow: Electron.BrowserWindow | null = null;
```

Comment:
Could you explain this change? At the top of the file, iit imports `BrowserWindow` explicitly, and `Electron` isn't defined.